systemd/cloud-init-hotplugd.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
The perms on this file need to change as mentioned in complaints seen from journalctl
```bash
Jul 09 03:09:23 test-hotplug systemd[1]: Configuration file /lib/systemd/system/cloud-init-hotplugd.service is marked executable. Please remove executable permission bits. Proceeding anyway.
```